dali/test/python/test_RN50_data_pipeline.py

```diff
@@ -135,15 +154,37 @@ def define_graph(self):
                     help='Use NHWC data instead of default NCHW')
 parser.add_argument('-i', '--iters', default=-1, type=int, metavar='N',
                     help='Number of iterations to run (default: -1 - whole data set)')
+parser.add_argument('--epochs', default=2, type=int, metavar='N',
+                    help='Number of epochs to run')
+parser.add_argument('--decoder_type', default='', type=str, metavar='N',
+                    help='split, roi, roi_split, cached (default: regular nvjpeg)')
+parser.add_argument('--cache_size', default=0, type=int, metavar='N',
+                    help='Cache size (in MB)')
+parser.add_argument('--cache_threshold', default=0, type=int, metavar='N',
+                    help='Cache threshold')
+parser.add_argument('--cache_type', default='none', type=str, metavar='N',
+                    help='Cache type')
+parser.add_argument('--reader_queue_depth', default=1, type=int, metavar='N',
+                    help='prefetch queue depth (default: 1)')
+parser.add_argument('--simulate_N_gpus', default=1, type=int, metavar='N',
+                    help='just here to test cached decoder')
```

Comment:
If this is intended to stay, maybe update the doc. Also something like `None` might make better default value (as I understood, if this is `1` `args.gpus` is used).

Reply:
will do